### ISSUE TITLE
[bug] Accessibility fixes

### ISSUE THREAD

**Issue by daria-github** [rainbow-me/rainbowkit#1092]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### RainbowKit Version

0.8.1

### wagmi Version

0.9.6

### Current Behavior

Hello! I'm working on implementing some WCAG updates to an app that uses Rainbow Kit, and seeing some issues within Rainbow Kit specifically related to color contrast, and also a couple on missing aria labels. I'm happy to open a PR on this if you'd like, but thought I'd raise the issue here first since it'd touch the `--rk-colors-profileForeground` var which may require discussion. These are coming up via the Axe extension.

### Expected Behavior

No color contrast or missing label errors.

### Steps To Reproduce

Run the Axe extension with the modal open.

**Comment by DanielSinclair** [comment]
@daria-github Your contribution would be greatly welcome! Feel free to document any issues you've noticed here as well. We have a theme system that makes modifying some of this easy, so we could create an additional high contrast theme or tweak the existing. https://github.com/rainbow-me/rainbowkit/blob/26e59998e4f59746b33ca015798377b4c590c4f6/packages/rainbowkit/src/themes/lightTheme.ts#L56

**Comment by DanielSinclair** [comment]
Will close this issue for now, but please do add additional documentation of the current contract issues if possible.
